https://git.ffmpeg.org/gitweb/ffmpeg.git/commitdiff/1231003c3c6d

libavfilter/vf_libplacebo.c:461:10: error: field designator 'force_icc_lut' does not refer to any field in type 'struct pl_render_params'
        .force_icc_lut = s->force_icc_lut,
         ^

--- doc/filters.texi.orig	2023-02-27 20:43:45 UTC
+++ doc/filters.texi
@@ -16242,9 +16242,6 @@ Disable built-in GPU sampling (forces LUT).
 @item disable_builtin
 Disable built-in GPU sampling (forces LUT).
 
-@item force_icc_lut
-Force the use of a full ICC 3DLUT for gamut mapping.
-
 @item disable_fbos
 Forcibly disable FBOs, resulting in loss of almost all functionality, but
 offering the maximum possible speed.
--- libavfilter/vf_libplacebo.c.orig	2023-02-27 20:43:45 UTC
+++ libavfilter/vf_libplacebo.c
@@ -458,7 +458,6 @@ static int process_frames(AVFilterContext *avctx, AVFr
         .polar_cutoff = s->polar_cutoff,
         .disable_linear_scaling = s->disable_linear,
         .disable_builtin_scalers = s->disable_builtin,
-        .force_icc_lut = s->force_icc_lut,
         .force_dither = s->force_dither,
         .disable_fbos = s->disable_fbos,
     };
@@ -847,7 +846,7 @@ static const AVOption libplacebo_options[] = {
     { "polar_cutoff", "Polar LUT cutoff", OFFSET(polar_cutoff), AV_OPT_TYPE_FLOAT, {.dbl = 0}, 0.0, 1.0, DYNAMIC },
     { "disable_linear", "Disable linear scaling", OFFSET(disable_linear), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1, DYNAMIC },
     { "disable_builtin", "Disable built-in scalers", OFFSET(disable_builtin), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1, DYNAMIC },
-    { "force_icc_lut", "Force the use of a full ICC 3DLUT for color mapping", OFFSET(force_icc_lut), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1, DYNAMIC },
+    { "force_icc_lut", "Deprecated, does nothing", OFFSET(force_icc_lut), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1, DYNAMIC | AV_OPT_FLAG_DEPRECATED },
     { "force_dither", "Force dithering", OFFSET(force_dither), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1, DYNAMIC },
     { "disable_fbos", "Force-disable FBOs", OFFSET(disable_fbos), AV_OPT_TYPE_BOOL, {.i64 = 0}, 0, 1, DYNAMIC },
     { NULL },
